tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from "@playwright/test"
+import { randomEmail, randomString } from "./helpers"
+
+const login = async (page: Page) => {
+  const email = randomEmail()
+  await page.fill('[name="traits.email"]', email)
+  await page.fill('[name="password"]', randomString())
+  await page.click('[value="password"]')
+  return email
+}
+
+test.describe("django-ory-cloud", () => {
+  for (const app of [
+    { url: "http://localhost:4000/", name: "django" },
+  ]) {
```

Comment:
You are looping over an array that contains only one element. You could write this simpler. Are you planning to add more URLs here in the future?

```suggestion
  const url = "http://localhost:4000/"
  const name = "django"
```

Reply:
You are right, this is an artifact of how the tests run in ory/docs, where they loop over a dozen URLs, one for each test. 
I think going with the "single workflow for each test" approach in this repo makes sense, and as such we can omit the loop, thanks :)